### DISCUSSION TITLE
Text as a link in ui.text()

### DISCUSSION THREAD

**Discussion by shamilh2o** [h2oai/wave#2056] [Q&A] [answered]
I am using an email and a set of domains to display in a ui.text().

![image](https://github.com/h2oai/wave/assets/101506482/9583869b-4a19-4b07-a5d4-c8848794be0d)

But it displays as below. Texts are taken as links. 

![image](https://github.com/h2oai/wave/assets/101506482/cfe7ec7b-7e7c-4d29-b243-585e218d89b5)

How can I simply display it as a plain text? Any suggestions? Thanks!
cc: @mturoci

**Comment by mturoci** [comment, marked as answer]
Hi @shamilh2o.

See this [SO post](https://stackoverflow.com/questions/25706012/how-do-i-prevent-auto-generated-links-in-the-github-wiki) to learn how to get around linkifying.

E.g.

```py
sample_markdown = '''=
martin@gmail<span>.</span>com
'''
```
